go/api/commands.go

```diff
@@ -49,4 +49,61 @@ type StringCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/get/
 	Get(key string) (string, error)
+
```

Comment:
There is no section in all docs which describes arguments and return value. Please add it for new commands and (could be in another PR) add for already existing ones.

Reply:
Done

Reply:
Thanks!